feature-libs/organization/unit-order/components/ng-package.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "../../../node_modules/ng-packagr/ng-package.schema.json",
```

need to go up one more folder 'level'

```suggestion
  "$schema": "../../../../node_modules/ng-packagr/ng-package.schema.json",
```